web_external/stylesheets/body/layersPanel.styl

```diff
@@ -6,8 +6,11 @@
 
     li
         border 0
-        margin 2px
-        padding 2px
+        margin 0
+        padding 8px 2px
```

Comment:
I have no objection to this, though I think it looks better with less padding (`padding 4px 0px`, for instance).